check.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Check Constraint
+summary: The Check constraint specifies that the column value must satisfy a Boolean expression.
+toc: false
+---
+
+The Check [constraints](constraints.html) specifies that the column value must satisfy a Boolean expression.
+
+<div id="toc"></div>
+
+## Details
+
+- Check constraints requires that the column values satisfy a Boolean expression within the constraint. The expression must evaluate to `TRUE` (or *NULL*) for every row affected by an `INSERT` or `UPDATE` statement. The DML statement will fail if the condition evaluates to `FALSE` for any row.
+- Check constraints may be specified at the column or table level and can reference other columns within the table. Internally, all column level Check constrints are converted to table level constraints so they can be handled in a consistent fashion.
+- You can have multiple Check constraints on a single column but ideally these should be combined using the logical operators. So, for example, 
```

Comment:
Yeah––it's running two independent checks and combining the results vs. a single check. Just computational efficiency.